site/docs/foundations/characteristics/index.mdx

```diff
@@ -6,42 +6,46 @@ sidebar:
   priority: 1
 ---
 
-## About Characteristics
+A design token stores design decisions in a single place, enabling teams to use them to create consistent experiences.
 
-Characteristic design tokens are a level of abstraction sitting between the foundational tokens and the component-specific tokens.
+The Salt Design System uses three tiers of tokens. From most generalized to most specific, they are:
 
-Each grouping of characteristics (also known in other design systems as _semantics_) is a set of CSS variables which encapsulates a specific concept. These tokens can then be reused by multiple components where necessary, allowing for consistency across design and logical, reusable patterns.
+1. Foundational tokens
+2. Characteristic design tokens (characteristics)
+3. Component-specific tokens
 
-### What are characteristics?
+Characteristics form sets of Cascading Style Sheet (CSS) variables that define a visual attribute, such as text or colors denoting status. Components can then reuse them, allowing for design consistency and logical patterns. They help to define the rules that style each component and aid the cohesiveness of the overall user interface.
 
-Characteristics are semantic groupings of visual attributes that are applied across relevant components. They drive consistency and support theming across complex libraries. They help to define the rules that style each component and aid the cohesiveness of the UI.
+## Benefits
 
-### Why characteristics?
+For designers, characteristics enable visual changes without impacting the efficiency or effectiveness of components.
 
-From a design perspective, there is a need for aesthetic flexibility without decreasing the efficiency of build or the effectiveness of the components when implemented. From a dev perspective, it provides an easy way to reuse the same values, without the overhead of having to remember details such as hex values. Any time you want to tweak the value of a characteristic, you only have to do it in one place for it to reflect everywhere.
+For developers, they offer an easy way to reuse the same values. It creates a single point of reference for commonly used values, which means you only have to update in one place for it to reflect everywhere.
 
 Characteristics offer the following benefits:
```

Comment:
we got rid of bullets for below points, but added for the bullets below? @mikearildbrown

Reply:
sorry, i meant we got rid of the full stops on these bullet points, but added them to the below ones

Reply:
below, bullet point full stops removed end of each sentence

- Ability to theme the Salt inline with third-parties or CIB sub brand guidelines.	- Ability to theme Salt in line with third parties or sub-brand guidelines
- Increase consistency across all components, with the knock-on effect of improving the overall user experience e.g. reduced cognitive load.	- Increased consistency across all components, improving the overall user experience, e.g., reduced cognitive load
- Simplified design of new components and patterns e.g. drag-and-drop behavior.	- Simplified design of new components and patterns e.g., drag-and-drop behavior
- Streamlined evolution – a single change to a characteristic will automatically cascade down through all related components within the library.	- Streamlined design changes, e.g., a characteristic tweak automatically applying to all related components

line ~36, full stops added
1. Always begin with `--salt-`.
2. Follow with the foundation name, e.g., `--salt-actionable-`, `--salt-text-`, `--salt-focused-`.
3. Append the variant if applicable, e.g., `--salt-actionable-cta-`, `--salt-text-h2-`.
4. Append the attribute, e.g., `--salt-actionable-cta-background`, `--salt-text-h2-fontSize`, `--salt-focused-outlineColor`.
5. Append the state if applicable, e.g., `--salt-actionable-cta-background-hover`.